### PR TITLE
fix: adjust the obsidian docs page

### PR DESCRIPTION
**Pull request by caleb-at-pieces**
closes https://github.com/pieces-app/documentation/issues/160

preview link: https://docs.pieces.app/~161/extensions-plugins/obsidian

- brand unification (Pieces for Developers Obsidian Plugin)
- remove incorrect information
- add sections to install instructions on pfd and pos
- adjust the getting started link

### CODE REVIEW

File: docs/extensions-plugins/obsidian.mdx [hunk 1]
```diff
@@ -103,51 +103,50 @@ export const HeaderLogoImage = (props) => {
...
+Installation instructions for the latest Pieces for Developers Obsidian Plugin release:
 
-1. Make sure you have community plugins enabled in Obsidian.
-2. Search for **Pieces for Developers** in the Obsidian community plugin store.
-3. Click `install` on the Pieces for Developers plugin listing.
-4. After the plugin is downloaded and installed, click `enable` to start the plugin.
-5. Now you can get started with the plugin! Just click the Pieces icon on the left, and start the magic!
+1. Make sure you have Pieces OS installed
+  a. Pieces OS is the core platform dependency for all Pieces applications. It is the backend that powers all our services, including the plugins and the Desktop App.
+2. (optional) Install the Pieces for Developers Desktop App for the fully fledged experience utilizing all of Pieces for Developers' AI features, further boosting your productivity.
```
Comment (mack-at-pieces) [inline docs/extensions-plugins/obsidian.mdx:112]: ...for the full experience utilizing...

File: docs/extensions-plugins/obsidian.mdx [hunk 2]
```diff
@@ -103,51 +103,50 @@ export const HeaderLogoImage = (props) => {
...
-[Installation Instructions](https://github.com/pieces-app/obsidian-pieces#install-instructions) for the latest Pieces for Obsidian
-release:
+Installation instructions for the latest Pieces for Developers Obsidian Plugin release:
 
-1. Make sure you have community plugins enabled in Obsidian.
-2. Search for **Pieces for Developers** in the Obsidian community plugin store.
-3. Click `install` on the Pieces for Developers plugin listing.
-4. After the plugin is downloaded and installed, click `enable` to start the plugin.
-5. Now you can get started with the plugin! Just click the Pieces icon on the left, and start the magic!
+1. Make sure you have Pieces OS installed
```
Comment (mack-at-pieces) [inline docs/extensions-plugins/obsidian.mdx:110]: Is it possible to add a hyperlink here to the appropriate page in the docs for them to get Pieces OS?

File: docs/extensions-plugins/obsidian.mdx [hunk 3]
```diff
@@ -103,51 +103,50 @@ export const HeaderLogoImage = (props) => {
...
+## Find your Snippets
 
-**To save a Snippet:**
-
-- Highlight the text, right-click, and select "Save to Pieces."
-- Click the Pieces button within any code block.
-- Highlight the desired code and use a dedicated keyboard shortcut.
+- Click on the Pieces icon in the ribbon menu to open the Pieces view.
+- Using the sort by dropdown menu, sort your snippets by **recent** or by **language**.
+- Using the searchbox at the top of the view, earch your snippets to save time leafing through your markdown files.
```
Comment (mack-at-pieces) [inline docs/extensions-plugins/obsidian.mdx:123]: Small typo here - "...search your snippets..."

File: docs/extensions-plugins/obsidian.mdx [hunk 4]
```diff
@@ -192,15 +191,15 @@ Pieces adds **embedded buttons** into your markdown files to help you save, copy
...
+One of the most significant features of this plugin is the Pieces Copilot, an artificial intelligence assistant designed to assist with your personal knowledge management.
 The Copilot learns from the notes in your vault and adapts to the context of what you're working on. To try out this impressive technology: 
 
-- Ensure Pieces OS is open and updated to at least 5.1.2.
-- To view the Copilot, open the Pieces plugin and click the bot logo on the top slider to switch to Copilot.
-- The Copilot will use your entire vault as context for the chatbot. This means you can query your Obsidian vault with plain text and we will semantically match your question to any of your relevant notes.
+- Ensure Pieces OS is open and updated.
+- To view the Pieces Copilot, open the Pieces for Developers plugin and click the bot logo on the top slider to switch to Copilot.
+- The Pieces Copilot will use your entire vault as context for the chatbot. This means you can query your Obsidian vault with plain text and we will semantically match your question to any of your relevant notes.
```
Comment (mack-at-pieces) [inline docs/extensions-plugins/obsidian.mdx:199]: Do you want to also say that this is customizable?

File: docs/extensions-plugins/obsidian.mdx [hunk 5]
```diff
@@ -260,24 +257,24 @@ To double-check that Pieces OS is running, make sure that the Pieces logo is pre
 If you need to download and install Pieces OS, please <a target="_blank" href="https://docs.pieces.app/overview">visit this link</a>.
 
 <h3>
-  If the Obsidian plugin still isn't working for you, please make sure you have the
+  If the Pieces for Developers Obsidian plugin still isn't working for you, please make sure you have the
```
Comment (mack-at-pieces) [inline docs/extensions-plugins/obsidian.mdx:260]: Would just make sure there's not an awkward linebreak here

**Review by mack-at-pieces — COMMENTED** [review]
Def better!

File: docs/extensions-plugins/obsidian.mdx (hunk 2 above)
Comment (caleb-at-pieces) [reply docs/extensions-plugins/obsidian.mdx:110]: The best link I see is this: https://docs.pieces.app/~161/installation-getting-started/what-am-i-installing#pieces-desktop-app--pieces-os

File: docs/extensions-plugins/obsidian.mdx (hunk 4 above)
Comment (caleb-at-pieces) [reply docs/extensions-plugins/obsidian.mdx:199]: - The Pieces Copilot is also highly customizable, as a user you can select which notes you would like the Pieces Copilot to use as context, as well as selecting which LLM runtime you would like to use, local or cloud.

File: docs/extensions-plugins/obsidian.mdx (hunk 2 above)
Comment (mack-at-pieces) [reply docs/extensions-plugins/obsidian.mdx]: That's perfect

File: docs.json
Comment (jordan-pieces) [inline docs.json:1]: @caleb-at-pieces thank you so much for updating all of these

**Review by jordan-pieces — APPROVED** [review]

**Pull request merged**